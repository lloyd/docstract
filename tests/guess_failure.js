/**
 * This module has an unguessable documentation block
 */

/**
 * WTF am I trying to document here?
 */
