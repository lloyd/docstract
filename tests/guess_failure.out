{
  "args": [
    "Can't determine what this block documents (from @function, @module, @property, @class) at line 5"
  ], 
  "exception_type": "<type 'exceptions.RuntimeError'>", 
  "stack": [
    [
      "<module>", 
      "got = ds.extractFromFile(os.path.join(testDir, test + \".js\"))"
    ], 
    [
      "extractFromFile", 
      "data = self.extract(contents)"
    ], 
    [
      "extract", 
      "self._analyzeBlock(block, context, firstBlock, data, lineStart, line)"
    ], 
    [
      "_analyzeBlock", 
      "guessedType = self._guessBlockType(firstBlock, codeChunk, thisContext, parseData['tagData'].keys())"
    ], 
    [
      "_guessBlockType", 
      "raise RuntimeError(\"Can't determine what this block documents (from %s)\" % \", \".join(possibilities))"
    ]
  ]
}
